example/spring-data-mongodb/src/test/kotlin/com/github/inflab/example/spring/data/mongodb/extension/AtlasTestConstructorExtension.kt

```diff
@@ -47,6 +51,11 @@ internal object AtlasTestConstructorExtension : ConstructorExtension {
                     "The parameter type of repository constructor must be MongoTemplate but ${repositoryParameterClass.qualifiedName} from ${parameterClass.qualifiedName}"
                 }
 
+                val database = repositoryParameter.annotations.find { it is Database } as Database?
+                if (database != null) {
+                    return@associateWith getMongoTemplate(getConnectionString(database.value))
+                }
```

Comment:
To remove warning from `return@associateWith`. 
we can use `let` scope function.

```suggestion
                repositoryParameter.findAnnotation<Database>()?.let {
                    getMongoTemplate(getConnectionString(it.value))
                } ?: mongoTemplate
```